test/ControlThreadTests.hs

```diff
@@ -0,0 +1,71 @@
+{-# LANGUAGE OverloadedStrings #-}
```

Comment:
Add some module level documentation. As in one control thread per torrent or something like that